docs/en/status-communities/understand-token-requirements-in-communities.md

```diff
@@ -0,0 +1,35 @@
+---
+id: 490
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Understand token requirements in communities
+
+<!--
+image
+-->
+
+In Status, there are two kinds of communities: public and private. If you wish to join a private community, you need to meet the community owner token requirements. To maintain access to a private community, hold the necessary tokens at all times.
+
+Community owners can configure token requirements for their communities. To join a community you need to meet its token requirements. For example, a community owner can set up 10 SNts as a requirement to join a community. So you need to have a minimum of 10 SNTs to join.
+
+The Status app classifies tokens into three different categories: assets, collectibles, and ENS (Ethereum Name Service) names. Community owners can configure any combination of these tokens as a requirement to join their communities.
+
+!!! note
+    When you join a private community, the token requirements are not a form of payment or transfer. They are simply used for verification purposes, and your tokens remain in your possession
```

Comment:
'possession.'
A full stop is missing :)

Reply:
When you join a private community, the token required to join are not transferred to anyone. They remain on your account.